doc/manuals/admin/database_model.md

```diff
@@ -236,8 +236,7 @@ Fields:
     below (we think they're quite straightforward)
 -   **count**: the number of notifications sent associated to
     the subscription.
--   **format**: the format to use to send notifications, either "XML"
-    or "JSON".
+-   **format**: the format to use to send notifications, or "JSON".
```

Comment:
Please use the following:

"the format to use to send notification, either "XML" or "JSON". However, note that XML has been deprecated in Orion 0.23.0 and that this field eventually will be removed"